downstream/modules/platform/ref-eda-controller-variables.adoc

```diff
@@ -1,215 +1,335 @@
-
-Path to the PostgreSQL SSL/TLS key file for {EDAName}.
-
-| `automationedacontroller_redis_host` | `eda_redis_host` | The Redis hostname used by {EDAcontroller}.
-
-| `automationedacontroller_redis_port` | `eda_redis_port` | The port used for the Redis host defined by `automationedacontroller_redis_host` for {EDAcontroller}. 
-
-| `automationedacontroller_rq_workers` | |  Number of Redis Queue (RQ) workers used by {EDAcontroller}. RQ workers are Python processes that run in the background.
-
-Default = (# of cores or threads) * 2 + 1
-
-| `automationedacontroller_ssl_cert` | `eda_tls_cert` | _Optional_
-
-Path to the SSL/TLS certificate file for {EDAName}.
-
-| `automationedacontroller_ssl_key` | `eda_tls_key` | _Optional_
-
-Path to the SSL/TLS key file for {EDAName}.
-
-| `automationedacontroller_user_headers` | `eda_nginx_user_headers` | List of additional NGINX headers to add to {EDAcontroller}'s NGINX configuration. 
-
-Default = empty list
-
-| `eda_node_type` | `eda_type` | _Optional_
-
-{EDAcontroller} node type.
-
-Default = `hybrid`
-
-| | `eda_debug` | {EDAcontroller} debug.
-
-Default = `false`
-
-| | `eda_event_stream_url` | {EDAcontroller} event stream URL.
-
-| | `eda_extra_settings` a| Defines additional settings for use by {EDAName} during installation.
+[cols="25%,25%,30%,10%,10%",options="header"]
+|===
+| RPM variable name | Container variable name | Description | Required or optional | Default
+
+| `automationedacontroller_activation_workers` 
+| `eda_activation_workers` 
+| Number of workers used for ansible-rulebook activation pods in {EDAName}.
+| Optional
+| RPM = (# of cores or threads) * 2 + 1. Container = `2`
+
+| `automationedacontroller_admin_email` 
+| `eda_admin_email` 
+| Email address used by Django for the admin user for {EDAName}.
+| Optional
+| `admin@example.com`
+
+| `automationedacontroller_admin_password` 
+| `eda_admin_password` 
+| {EDAName} administrator password. Use of special characters for this variable is limited. The password can include any printable ASCII character except `/`, `”`, or `@`.
+| Required
+|
+
+| `automationedacontroller_admin_username` 
+| `eda_admin_user` 
+| Username used to identify and create the administrator user in {EDAName}.
+| Optional
+| `admin`
+
+| `automationedacontroller_backend_gunicorn_workers`
+|
+| Number of workers for handling the API served through Gunicorn on worker nodes.
+| Optional
+| `2`
+
+| `automationedacontroller_cache_tls_files_remote`
+|
+| Denote whether the cache cert sources are local to the installation program (`false`) or on the remote component server (`true`).
+| Optional
+| `false`
+
+| `automationedacontroller_client_regen_cert`
+| 
+| Controls whether or not to regenerate {EDAName} client certificates for the platform cache. Set to `true` to regenerate {EDAName} client certificates.
+| Optional
+| `false`
+
+| `automationedacontroller_default_workers`
+|
```

Comment:
```suggestion
| `automationedacontroller_default_workers`
| `eda_workers`
```

Then remove the eda_workers entry for containerized at the end of the file (~ L330)